boards/clue_nrf52840/src/main.rs

```diff
@@ -0,0 +1,491 @@
+        ),
+    )
+    .finalize(components::gpio_component_buf!(nrf52840::gpio::GPIOPin));
+
+    //--------------------------------------------------------------------------
+    // LEDs
+    //--------------------------------------------------------------------------
+
+    let led = components::led::LedsComponent::new(components::led_component_helper!(
+        LedHigh<'static, nrf52840::gpio::GPIOPin>,
+        LedHigh::new(&base_peripherals.gpio_port[LED_RED_PIN]),
+        LedHigh::new(&base_peripherals.gpio_port[LED_WHITE_PIN])
+    ))
+    .finalize(components::led_component_buf!(
+        LedHigh<'static, nrf52840::gpio::GPIOPin>
+    ));
+
+    //--------------------------------------------------------------------------
+    // Buttons
+    //--------------------------------------------------------------------------
+    let button = components::button::ButtonComponent::new(
+        board_kernel,
+        components::button_component_helper!(
+            nrf52840::gpio::GPIOPin,
+            (
+                &base_peripherals.gpio_port[BUTTON_LEFT],
+                kernel::hil::gpio::ActivationMode::ActiveHigh,
+                kernel::hil::gpio::FloatingState::PullUp
+            ), // Left
+            (
+                &base_peripherals.gpio_port[BUTTON_RIGHT],
+                kernel::hil::gpio::ActivationMode::ActiveLow,
+                kernel::hil::gpio::FloatingState::PullUp
+            ) // Right
+        ),
+    )
+    .finalize(components::button_component_buf!(nrf52840::gpio::GPIOPin));
+
+    //--------------------------------------------------------------------------
+    // Deferred Call (Dynamic) Setup
+    //--------------------------------------------------------------------------
+
+    let dynamic_deferred_call_clients =
+        static_init!([DynamicDeferredCallClientState; 2], Default::default());
+    let dynamic_deferred_caller = static_init!(
+        DynamicDeferredCall,
+        DynamicDeferredCall::new(dynamic_deferred_call_clients)
+    );
+    DynamicDeferredCall::set_global_instance(dynamic_deferred_caller);
+
+    //--------------------------------------------------------------------------
+    // ALARM & TIMER
+    //--------------------------------------------------------------------------
+
+    let rtc = &base_peripherals.rtc;
+    rtc.start();
+
+    let mux_alarm = components::alarm::AlarmMuxComponent::new(rtc)
+        .finalize(components::alarm_mux_component_helper!(nrf52::rtc::Rtc));
+    let alarm = components::alarm::AlarmDriverComponent::new(board_kernel, mux_alarm)
+        .finalize(components::alarm_component_helper!(nrf52::rtc::Rtc));
+
+    //--------------------------------------------------------------------------
+    // UART & CONSOLE & DEBUG
+    //--------------------------------------------------------------------------
+
+    // Setup the CDC-ACM over USB driver that we will use for UART.
+    // We use the Arduino Vendor ID and Product ID since the device is the same.
```

Comment:
You are using different IDs than the nano33, so this comment should probably say Adafruit instead of Arduino?

Reply:
Updated, sorry :)